api/v1beta1/rollout_types.go

```diff
@@ -290,6 +497,17 @@ const (
 	RolloutPhaseDisabling RolloutPhase = "Disabling"
 )
 
+type FinalisingStepType string
+
+const (
+	FinalisingStepTypePreparing     FinalisingStepType = "Preparing"
```

Comment:
plz comment about each step and the common workflow that involves multiple steps